internal/configs/annotations_test.go

```diff
@@ -163,6 +167,59 @@ func TestMergeMasterAnnotationsIntoMinion(t *testing.T) {
 	}
 }
 
+func TestParseProxySetHeaderInputString(t *testing.T) {
+	t.Parallel()
+	headers1 := []string{"X-Forwarded-For"}
+	headers2 := []string{"ABC"}
+	headers3 := []string{"Test"}
+
+	headers := [][]string{headers1, headers2, headers3}
+	for _, header := range headers {
+		err := ParseProxySetHeader(header)
+		if err != nil {
+			t.Errorf("want nil, got error on valid input: %+v", header)
+		}
+	}
+}
+
+func TestParseProxySetHeaderInvalidInputString(t *testing.T) {
+	t.Parallel()
+	headers1 := []string{"X-Forwarded-For1"}
+	headers2 := []string{"ABC!"}
+	headers3 := []string{""}
+	headers4 := []string{" "}
+
+	headers := [][]string{headers1, headers2, headers3, headers4}
+	for _, header := range headers {
+		err := ParseProxySetHeader(header)
+		if err == nil {
+			t.Errorf("want error on input %+v, got nil", header)
+		}
+	}
+}
+
+// ParseProxySetHeader ensures that the string value contains only letters
+func ParseProxySetHeader(headers []string) error {
+	for _, header := range headers {
+		if err := ValidateProxySetHeader(header); err != nil {
+			fmt.Println("errored header = ", header)
```

Comment:
You can remove this print statement since each test that uses this function reports the header that caused the error.

Reply:
Removed.